core/network/src/consensus_gossip.rs

```diff
@@ -210,6 +210,11 @@ pub trait Validator<B: BlockT>: Send + Sync {
 	fn message_allowed<'a>(&'a self) -> Box<FnMut(&PeerId, MessageIntent, &B::Hash, &[u8]) -> bool + 'a> {
 		Box::new(move |_who, _intent, _topic, _data| true)
 	}
+
+	/// Produce a closure for filtering egress messages to a peer.
+	fn message_allowed_for_peer<'a>(&'a self) -> Box<FnMut(&[u8]) -> bool + 'a> {
+		Box::new(move |_data| true)
```

Comment:
I'm confused. For which peer? It's not a parameter to the function or the closure.

Reply:
Fixed, I didn't have very clear the code.